### PR TITLE
map overrideAccess on API update

### PR DESCRIPTION
**Pull request by Okhelifi**
## Issue

https://gravitee.atlassian.net/browse/APIM-2062

## Description

Currently we lose the overrideAccess attribute in entrypoint paths when we update an API

## Additional context

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-xemtvoflss.chromatic.com)

### CODE REVIEW

**Review by jgiovaresco — APPROVED** [review]
Shouldn't we need to add the mergify label `apply-on-master` ?

**Comment by Okhelifi** [comment]
> Shouldn't we need to add the mergify label `apply-on-master` ?

I was waiting for an approval before adding it but yes I add it now. Thanks for the review.

**Pull request merged**
